tests/Operation/WatchFunctionalTest.php

```diff
@@ -720,7 +720,7 @@ public function testInitialCursorIsNotClosed(): void
          * reports the cursor as alive. While the cursor ID is accessed through
          * ChangeStream, we'll need to use reflection to access the internal
          * Cursor and call isDead(). */
-        $this->assertNotEquals('0', (string) $changeStream->getCursorId(true));
+        $this->assertNotEquals(0, $changeStream->getCursorId());
```

Comment:
Would it make the test more robust if you don't rely on int cast?
```suggestion
        $this->assertNotEquals(new Int64(0), $changeStream->getCursorId());
```

Reply:
It doesn't make a difference, as the custom `Int64Comparator` will always handle these assertions, and it leverages the comparison behaviour implemented in `Int64` directly.